.env

```diff
@@ -28,3 +28,11 @@ POSTGRES_HOST=db
 #
 MAX_CONCURRENCY=80
 MIN_CONCURRENCY=10
+
+#
+# Cloud default django superuser account
+# This only applies if you are using the cloud deployment
+#
+DJANGO_CLOUD_SUPERUSER_USERNAME=cloudadmin
+DJANGO_CLOUD_SUPERUSER_EMAIL=cloudadmin@rengine.invalid
+DJANGO_CLOUD_SUPERUSER_PASSWORD=IxEXqBfOq2eQWlXzeMspMToDRKyXIS
```

Comment:
Is this necessary considering its not set for the default deployment method ?

Reply:
If you check the Makefile you'll see it runs an interactive script to have you set the superuser. This way of doing this lets us install everything completely unattended